contentcuration/contentcuration/frontend/shared/directives/mouse-move.js

```diff
@@ -1,14 +1,12 @@
+const listeners = new Map();
 export default {
-  inserted: (el, binding) => {
-    window.event = event => {
```

Comment:
`window.event` was getting used by all the directives for all the Vue component instances using it, which meant that all of these instances were interfering with each other.